packages/apollo-link-dedup/README.md

```diff
@@ -0,0 +1,51 @@
+# Dedup Link
+
+## Purpose
+An Apollo Link to deduplicate matching requests before making a request.
+
+*NOTE* This link is included by default when using apollo-client.

```

Comment:
What does this mean? So if I pass in an `HttpLink` then this still gets added to it?

Reply:
yep!

Reply:
Do we have any way to override that?